ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -427,23 +425,37 @@ void NGraphEncapsulateOp::ComputeUsingParallelExecutor(OpKernelContext* ctx) {
     tf_input_tensors.push_back(ctx->input(i));
   }
 
-  NGRAPH_VLOG(2) << "[PREFETCH] COMPUTE: Input: " << ctx->num_inputs()
-                 << " Values: " << tf_input_tensors[0].DebugString();
-  int step_id = ctx->step_id();
+  // Get ngraph executable,function and Pipelined Tensor Store
   ngraph::Event event_get_ng_item("GetExecutableAndTensors", "", "");
+  std::shared_ptr<ngraph::runtime::Executable> ng_exec;
+  std::string serialized_ng_function;
   shared_ptr<PipelinedTensorsStore> pipelined_tensor_store;
-  std::tuple<int, PipelinedTensorVector, PipelinedTensorVector> io_tensors;
-
-  // Get ngraph executable and inputs information and Pipelined tensors
   bool cache_hit;
-  std::string serialized_ng_function;
+
   OP_REQUIRES_OK(ctx, m_parallel_executor->GetExecutableFunctionAndTensors(
                           tf_input_tensors, ng_exec, serialized_ng_function,
                           pipelined_tensor_store, cache_hit));
+  NGRAPH_VLOG(2) << "CACHE HIT: " << PrintBool(cache_hit) << endl;
+  NGRAPH_VLOG(2) << " Step_ID: " << ctx->step_id();
+
+  NGRAPH_VLOG(2)
+      << "NGraphEncapsulateOp::Compute got ngraph executable for cluster id: "
+      << m_parallel_executor->GetNgraphClusterId();
+
+  event_get_ng_item.Stop();
+  ngraph::Event::write_trace(event_get_ng_item);
+
+  // Error check for pipelined tensors and pipeline depth
+  OP_REQUIRES(ctx, m_parallel_executor->GetTensorPipelineDepth() == 2,
```

Comment:
instead of calling m_parallel_executor->GetTensorPipelineDepth() 
two times, can you store its value in some variable and then use that.

Reply:
not storing in a variable coz its used only in error checking, not used anywhere else